stdlib/runtime/string.gr

```diff
@@ -46,42 +46,33 @@ enum StringList {
 primitive _RUNTIME_TYPE_METADATA_PTR: WasmI32 = "@heap.type_metadata"
 
 @unsafe
-let findTypeMetadata = (moduleId, typeId) => {
+let findTypeMetadata = typeId => {
```

Comment:
This function is now quite a bit less efficient since it traverses _all_ type information. It should be converted to something faster like a binary tree or a hash, though it's tricky to implement because the metadata itself lives in runtime land where arbitrary allocations are not available. Regardless, I felt that optimization didn't need to be included in this PR and could come separately.